src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc

```diff
@@ -110,7 +110,7 @@ void NaviStartStreamRequest::on_event(const event_engine::Event& event) {
       }
       if (hmi_apis::Common_Result::REJECTED == code) {
         LOG4CXX_INFO(logger_, "StartStream response REJECTED ");
-        SendRequest();
+        RetryStartSession();
```

Comment:
@Anatoliy-Leshin you're going to send one more `NaviStartStream` request when you call `RetryStartSession()` but not going to terminate existing one. What's the reason to call `RetryStartSession()` if `SendRequest()` performs exactly what you need (send NaviStartStream to HMI)?
And seems like this fix is not related to task specified in description. Please correct me if I'm not right.